bucket/mockery.json

```diff
@@ -1,14 +1,21 @@
 {
-    "version": "2.8.0",
+    "version": "2.9.4",
     "description": "A mock code autogenerator for Golang",
     "homepage": "https://github.com/vektra/mockery",
     "license": "BSD-3-Clause",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/vektra/mockery/releases/download/v2.8.0/mockery_2.8.0_Windows_x86_64.tar.gz",
-            "hash": "fb8e15fa71e121795ae664c23aa85aee4a66118b98584594052e914399b94c83"
+            "url": "https://github.com/vektra/mockery/releases/download/v2.9.4/mockery_2.9.4_Windows_x86_64.tar.gz",
+            "hash": "079f0143442f5f146fa7e318de6e646b90acdbc0118cacf7b67e7e2ec0b91809"
         }
     },
     "bin": "mockery.exe",
-    "checkver": "github"
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/vektra/mockery/releases/download/v$version/mockery_$version_Windows_x86_64.tar.gz"
+            }
```

Comment:
Mockery has a checksums.txt file

Could we use that for hashes?

https://github.com/vektra/mockery/releases/download/v2.9.4/checksum.txt